spec/compiler/type_inference/did_you_mean_spec.cr

```diff
@@ -242,4 +242,23 @@ describe "Type inference: did you mean" do
       end
       ), "did you mean @@foobar"
   end
+
+  it "suggests a better alternative to logical operators (#2715)" do
+    message = "undefined method 'and'"
+    message = " (did you mean '&&'?)".colorize.yellow.bold.to_s
```

Comment:
is there any problem using the colorize here? I tried to avoid but didn't realize how to, so kept to make the test pass, any suggestion is welcome.